### PR TITLE
Update rendering

### LINKED ISSUE #56: 403 errors on preview and facttable
Why are 403 errors handled separately than other 403 error in the report editor?
See https://www.dropbox.com/s/stk9u1j4o2znsql/Screenshot%202014-07-18%2004.41.41.png

### LINKED ISSUE #102: "show text" link in fact table view does not work
report_editor_and_rent_lease_concepts_xlsx__1_page_

### LINKED ISSUE #105: solid line at the bottom of the table missing (if exceeds visible browser area)
Safari

report_editor

### PR DESCRIPTION
**Pull request by Alexander-Kreutz**
integrates the newest version of the layouter and should fix these issues:
fixes #105
fixes #102
fixes #56

### CODE REVIEW

**Comment by wcandillon** [comment]
@Alexander-Kreutz looks good. What could be tested in this branch that would fail in master? I need to check how we want to do the error handling in the report editor (what we want to do on the alter event).

File: bower.json [hunk 1]
```diff
@@ -36,12 +36,13 @@
     "angular-cache": "2.3.0",
     "bootstrap": "~3.1.1",
     "accounting": "~0.3.2",
-    "layoutmodel-renderer-angular" : "https://github.com/28msec/layoutmodel-renderer-angular.git#721dc155fa6bf8f9b14bf2d61bd793afa2d9d63b",
+    "layoutmodel-renderer-angular": "https://github.com/28msec/layoutmodel-renderer-angular.git#74ff25e12e4e2b116e30b187e0f2db42c916f7b4",
```
Comment (wcandillon) [inline bower.json]: This commit is not in master. Isn't that dangerous somehow? I think it fine but I'm wondering if there is a better way to do that.

Comment (Alexander-Kreutz) [inline bower.json]: I guess this defines exactly the version it should run with. The hash should still work after the corresponding branch in the renderer has been merged into master.

**Comment by Alexander-Kreutz** [comment]
@wcandillon only the "show text" issue #102 could be tested. The bad thing is I do not know how to create a report that contains such a link in the facttable. We need better information in the issues itself on how to produce a bug. (A screenshot is nice but does not help in producing a corresponding report)

**Comment by MatthiasBrantner** [comment]
Alexander

You can create a report with a concept called NatureOfOperations with synonyms:

```
• us-gaap:NatureOfOperations
• us-gaap:BusinessDescriptionAndBasisOfPresentationTextBlock
• us-gaap:OrganizationConsolidationAndPresentationOfFinancialStatementsDisclosureTextBlock
• us-gaap:OrganizationConsolidationBasisOfPresentationBusinessDescriptionAndAccountingPoliciesTextBlock
```

As filters, you can use the latest American Express filing.

HTH

Matthias

**Comment by wcandillon** [comment]
@Alexander-Kreutz do we need to close it? Is master more recent than this PR somehow?

**Comment by Alexander-Kreutz** [comment]
The branch removes the "Label" choice in the spreadsheet and does the 
errors as alerts instead of red boxes. I think we should merge it.

**Comment by wcandillon** [comment]
@Alexander-Kreutz Could you try to merge master into this branch? I tried to do it but it was tricky. If you have issues we could try to do it together.

**Comment by Alexander-Kreutz** [comment]
Done

**Comment by wcandillon** [comment]
yes that's good :)

**Pull request merged**